apollo-router/src/services/router.rs

```diff
@@ -66,6 +66,33 @@ impl From<(http::Request<Body>, Context)> for Request {
     }
 }
 
+/// Helper type to conveniently construct a body from several types used commonly in tests.
+///
+/// It's only meant for tests, as the "real" router should create bodies explicitly accounting for
+/// streaming, size limits, etc.
+pub struct IntoBody(Body);
```

Comment:
can we use `#[cfg(test)]` around this? And should this be `pub` or `pub(crate)`?

Reply:
It's being used in integration tests. I'll update the comment to make that clearer.

Reply:
done: 27219bb5a8129882705f17888f745a89be5d67dd